Package.swift

```diff
@@ -52,8 +52,8 @@ let package = Package(
                 path: "TestTools/StreamChatTestTools",
                 exclude: ["Info.plist"],
                 resources: [
-                        .copy("Fixtures/JSONs"),
-                        .copy("Fixtures/Other"),
+                        .process("Fixtures/JSONs"),
+                        .process("Fixtures/Other"),
                         .process("Fixtures/Images")
```

Comment:
Maybe we could just simply process the parent folder. Worth to test against testing branch ;)

```suggestion
                        .process("Fixtures")
```

Reply:
@bielikb, thanks, that's cleaner and it still works 👍